styleguide/source/_patterns/02-molecules/inline-bio.twig

```diff
@@ -8,8 +8,6 @@
   {% endif %}
 
   <div class="ama__inline-bio__text">
-    {% include "@atoms/heading.twig" %}
```

Comment:
This is removing the name as the heading. This needs to be put back.

Reply:
you're right. I assumed this was for the position and title.